cloud/kubernetes/README.md

```diff
@@ -177,14 +172,25 @@ Along with our StatefulSet configuration, we expose a standard Kubernetes servic
 that offers a load-balanced virtual IP for clients to access the database
 with. In our example, we've called this service `cockroachdb-public`.
 
-Start up a client pod and open up an interactive, (mostly) Postgres-flavor
-SQL shell using:
+In insecure mode, start up a client pod and open up an interactive, (mostly) Postgres-flavor
+SQL shell:
 
-```console
-$ kubectl run -it --rm cockroach-client --image=cockroachdb/cockroach --restart=Never --command -- ./cockroach sql --host cockroachdb-public
+```shell
+$ kubectl run cockroachdb -it --image=cockroachdb/cockroach --rm --restart=Never \
+    -- sql --insecure --host=cockroachdb-public
 ```
 
-**WARNING**: there is no secure mode equivalent of doing this (yet).
+In secure mode, use our `cockroachdb-client-secure.yaml` config to launch a pod that runs indefinitely with the `cockroach` binary inside it:
+
+```shell
+kubectl create -f cockroachdb-client-secure.yaml
+```
+
+Check and approve the CSR for the pod as described above, and then get a shell to the pod and run:
+
+```shell
+kubectl exec -it <pod name> -- ./cockroach sql --certs-dir=/cockroach-certs --host=cockroachdb-public
```

Comment:
If we make this a raw `Pod` object rather than a `Deployment`, we'll know the name in advance and can hard-code it in the instructions. Assuming it's ok with you, I'm going to push an update to your branch since that'll be easier than giving you the entire yaml in a comment.

Reply:
Sounds good, @a-robinson. Thanks!

Reply:
Done.